src/CTA.Rules.Actions/Csharp/ClassActions.cs

```diff
@@ -22,28 +22,30 @@ public class ClassActions
         {
             ClassDeclarationSyntax RemoveBaseClass(SyntaxGenerator syntaxGenerator, ClassDeclarationSyntax node)
             {
-                SeparatedSyntaxList<BaseTypeSyntax> currentBaseTypes = node.BaseList.Types;
-                SeparatedSyntaxList<BaseTypeSyntax> newBaseTypes = new SeparatedSyntaxList<BaseTypeSyntax>();
-
-                foreach (var baseTypeSyntax in currentBaseTypes)
+                if (node.BaseList != null)
```

Comment:
Do you know why or in what situation the BaseList is null? Is it related to Partial class or anything?

Reply:
BaseList is null when there is no base class.